prettyprinter/test/Testsuite/Main.hs

```diff
@@ -111,6 +114,17 @@ fusionDoesNotChangeRendering depth
             , "Fused:"
             , indent 4 (pretty renderedFused) ]
 
+groupLayoutEqualsSimpleGroupLayout :: Property
+groupLayoutEqualsSimpleGroupLayout = forAllShow (arbitrary :: Gen (Doc Int)) (show . diag) (\doc ->
+    forAll arbitrary (\layouter ->
+        let grouped = group doc
+            groupedSimple = simpleGroup doc
+            groupedLayedOut = layout layouter grouped
+            groupedSimpleLayedOut = layout layouter groupedSimple
+        in counterexample ("Grouped: " ++ (show . diag) (grouped))
+            (counterexample ("Grouped (Simple) " ++ (show . diag) (groupedSimple))
```

Comment:
This nested `counterexample` is a bit surprising IMHO – I'd prefer something similar to `mkCounterexample` above.

Reply:
^^ Agree, just took a shortcut tbh ;)